SayTheirNames/Resources/Base.lproj/Localizable.strings

```diff
@@ -130,3 +130,24 @@
 
 /* loco:5ee2ce86663b704aa41688f2 */
 "sign" = "Sign this petition";
+
```

Comment:
I assume those were part of a different effort to add new keys ?

Reply:
indeed!